packages/@aws-cdk/aws-apigatewayv2/lib/http/authorizer.ts

```diff
@@ -110,6 +166,10 @@ export class HttpAuthorizer extends Resource implements IHttpAuthorizer {
       throw new Error('jwtAudience and jwtIssuer are mandatory for JWT authorizers');
     }
 
+    if (props.type === HttpAuthorizerType.LAMBDA && (typeof props.enableSimpleResponses === 'undefined' || !props.authorizerUri)) {
+      throw new Error('enableSimpleResponses, and authorizerUri are mandatory for Lambda authorizers');
```

Comment:
The documentation states that the default behaviour is `IAM` responses, so it would make sense to assume a default `enableSimpleResponses = true`. Throwing this error if `enableSimpleResponses` is not specified seems an unnecessary restriction.

Reply:
It's not unnecessary, this value is irrelevant for someone using JWT so there's no point in setting it default. Remember Authorizer is really a base class and that the user will interact with `HttpLambdaAuthorizer` - https://github.com/aws/aws-cdk/blob/1b0adce87c38996dff3133fc44115b60cdf116c9/packages/%40aws-cdk/aws-apigatewayv2-authorizers/lib/http/lambda.ts#L71

We could apply this suggestion there instead.

Reply:
I agree with @kbanman here. 

If CF's default for this property for request authorizers is IAM (which seems like a good default), we should remove this validation and just document this in the '@default' section of this property.

Reply:
Made some changes, let me know what you think